src/deposit.cairo

```diff
@@ -374,10 +379,22 @@ mod Deposit {
                 IAirdropDispatcher { contract_address: airdrop_addr }.claim(claim_data, proof),
                 'Claim failed'
             );
-            // TODO: Add transfer to the Treasury
+
+            let strk = ERC20ABIDispatcher { contract_address: STRK_ADDRESS.try_into().unwrap() };
+            let zk_market = self.zk_market.read();
+            let half_of_claim = claim_data.amount / 2; // u128 integer division, rounds down
```

Comment:
Please, pay attention, not half of it. 
Imagine that user will have reward as 7% APY, you will get user half of it and we will lose 1.5% of revenu.
I think the fastest way just divide into 8 or 10 instead of 2. instaed of calculating collateral amount -> convert it to usdc value -> calculate time -> convert starknet token to usdc -> calculate user part.